_tasks/test_browser/server.ts

```diff
@@ -0,0 +1,15 @@
+import { serveDir } from "https://deno.land/std@0.172.0/http/file_server.ts"
+import { serve } from "https://deno.land/std@0.172.0/http/server.ts"
+
+import { PORT, STATIC_DIR } from "./constants.ts"
+
+const ac = new AbortController()
+
+const handler = async (req: Request): Promise<Response> =>
+  await serveDir(req, { fsRoot: STATIC_DIR })
+
+export const startServer = () => {
+  serve(handler, { port: PORT, signal: ac.signal })
+}
+
+export const stopServer = () => ac.abort()
```

Comment:
One cannot reuse an abort controller instance, so exposing this––in this way––is moot. Like I say above, let's avoid unnecessary aliasing. If it's no reused, air on the side of less code.